pkg/kontainerdrivermetadata/metadata_handler.go

```diff
@@ -59,31 +57,27 @@ var (
 	fileMapData = map[string]bool{}
 )
 
-func Register(ctx context.Context, management *config.ManagementContext) {
-	mgmt := management.Management
+func Register(ctx context.Context, wCtx *wrangler.Context) {
 
 	m := &MetadataController{
-		SystemImagesLister:   mgmt.RkeK8sSystemImages("").Controller().Lister(),
-		SystemImages:         mgmt.RkeK8sSystemImages(""),
-		ServiceOptionsLister: mgmt.RkeK8sServiceOptions("").Controller().Lister(),
-		ServiceOptions:       mgmt.RkeK8sServiceOptions(""),
-		NamespacesLister:     management.Core.Namespaces("").Controller().Lister(),
-		AddonsLister:         mgmt.RkeAddons("").Controller().Lister(),
-		Addons:               mgmt.RkeAddons(""),
-		SettingLister:        mgmt.Settings("").Controller().Lister(),
-		Settings:             mgmt.Settings(""),
+		SystemImagesController:   wCtx.Mgmt.RkeK8sSystemImage(),
+		ServiceOptionsController: wCtx.Mgmt.RkeK8sServiceOption(),
+		NamespacesController:     wCtx.Core.Namespace(),
```

Comment:
This is not a blocker, but the naming of fields feels unusual to me. Is there any particular reason to have `Controller` suffix here? I don't see this anywhere in our handlers.